### PR TITLE
feat: add prettier

### PR DESCRIPTION
**Pull request by Olyno**
# Description

This pull request simply adds and configure ``prettier`` to have a consistent code format.

### CODE REVIEW

File: .prettierrc [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+  "arrowParens": "avoid",
+  "bracketSpacing": true,
+  "jsxSingleQuote": false,
+  "quoteProps": "as-needed",
+  "semi": false,
```
Comment (HarryET) [inline .prettierrc]: I like to have semi colons on

File: .prettierrc [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+  "tabWidth": 2,
+  "singleQuote": true
```
Comment (HarryET) [inline .prettierrc]: I prefer double quote, any comment here from @VictorPeralta?

**Review by HarryET — CHANGES_REQUESTED** [review]

**Review by HarryET — CHANGES_REQUESTED** [review]
Prettier needs to be re-run

**Comment by Olyno** [comment]
I also added the missing ``.next`` folder in the format ignore

**Review by VictorPeralta — APPROVED** [review]

**Review by HarryET — APPROVED** [review]

**Pull request merged**